x-pack/elastic-agent/pkg/agent/application/info/agent_metadata.go

```diff
@@ -126,8 +130,10 @@ func (i *AgentInfo) ECSMetadata() (*ECSMeta, error) {
 	return &ECSMeta{
 		Elastic: &ElasticECSMeta{
 			Agent: &AgentECSMeta{
-				ID:      i.agentID,
-				Version: release.Version(),
+				ID:            i.agentID,
+				Version:       release.Version(),
+				Snapshot:      release.Snapshot(),
+				BuildOriginal: release.Info().String(),
```

Comment:
This is probably a great place to put if the agent is also upgradable.

Should we add `Upgradable: false` for now, then replace it with the correct check once #21206 lands.